digits/job.py

```diff
@@ -132,6 +133,9 @@ def path_is_local(self, path):
     def name(self):
         return self._name
 
+    def notes(self):
+        return self._notes
```

Comment:
Let's add a check to make sure that `_notes` exists and return `None` if not. For older jobs I'm getting this error:

    AttributeError: 'ImageClassificationModelJob' object has no attribute '_notes'